licenses.json

```diff
@@ -9119,7 +9121,7 @@
     "publisher": "Web SDK Customer Support",
     "email": "web-sdk@onfido.com",
     "url": "https://github.com/onfido",
-    "path": "onfido-sdk-ui",
+    "path": "/Users/sher.bhachu/Documents/Repo/Public/onfido-sdk-ui",
```

Comment:
This script is a bit annoying and it inserts your local repo path. You have to manually delete it.

Reply:
Thanks, I didnt even notice that!